controller/base_controller.go

```diff
@@ -6,6 +6,17 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
+var (
+	// maxRetries is the number of times a deployment will be retried before it is dropped out of the queue.
+	// With the current rate-limiter in use (5ms*2^(maxRetries-1)) the following numbers represent the times
+	// a deployment is going to be requeued:
+	//
+	// 5ms, 10ms, 20ms
+	maxRetries = 3
+
+	maxRetriesOnAcquireLockError = 16
```

Comment:
It's a customized threshold for backup_controller only. It'd better put in backup_controller.go